zeppelin-web/src/app/tabledata/tabledata.js

```diff
@@ -64,12 +62,10 @@ export default class TableData extends Dataset {
             col = valueOfCol;
           }
           cols.push(col);
-          cols2.push({key: (columnNames[i]) ? columnNames[i].name : undefined, value: col});
```

Comment:
you need to handle the unset columnNames[i] case in visualization-table.js?

Reply:
No. I deleted these lines because `array` is not used anywhere.

Reply:
I mean, it seems `columnNames[i]` can be null or unset.

Reply:
Yes. It would be correct to use `j` instead of `i`